lib/locations/locations.go

```diff
@@ -128,6 +134,43 @@ func expandLocations() error {
 	return nil
 }
 
+// ListExpandedPaths returns a machine-readable mapping of the currently
+// configured locations.  Unexpanded entries are filtered out.
+func ListExpandedPaths() map[string]string {
+	res := make(map[string]string, len(locations))
+	for key, path := range baseDirs {
+		res["baseDir-"+string(key)] = path
```

Comment:
I'd prefer two maps for base directories and paths/files over this prefix.

Reply:
Also for the API? I think it makes the whole thing needlessly complicated when there is a `paths` property and then a separate `baseDirs` property, or one `paths` with two nested sub-objects (called what?).

All the keys in that map are well-defined strings, so this prefix is just a part of the keyspace.  It might look cleaner when making the whole thing a structure as you suggested.  Let me see what I can come up with there.  But do you think all the calling places should be changed to access a specific object instead of just calling a function from that module?  I'd rather not change that in principle.

Reply:
lets leave as is